backend/routes/profile.py

```
@@ -0,0 +1,109 @@
+from typing import List
+from fastapi import APIRouter, Depends, HTTPException, UploadFile, File, Form
+from sqlalchemy.orm import Session
+import models.user as models
```

Ruff (0.8.2)

`models.user` imported but unused

Remove unused import: `models.user`

(F401)